patches/tModLoader/Terraria/ModLoader/IO/WallEntry.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Terraria.ID;
+
+namespace Terraria.ModLoader.IO
+{
+	internal class WallEntry : ModEntry
+	{
+		public static Func<TagCompound, WallEntry> DESERIALIZER = tag => DeserializeTag(tag);
+
+		public static WallEntry DeserializeTag(TagCompound tag) {
```

Comment:
This *probably* could be generalized and moved up to ModEntry, but I missed that when doing it.